DarkFlippers/unleashed-firmware
# SubGhz Analyzer PRECISE hstep fix

# What's new

- Fix steps for PRECISE. Previously was using the default 5 MHz step size
- Give ULTRANARROW #.# labels, since it can increment in 0.5 MHz intervals

# Verification 

- Use PRECISE mode and move left and right by 100 kHz
- Use ULTRANARROW mode and note that the labels now show the first decimal place

# Checklist (For Reviewer)
